common/src/main/java/dev/ftb/mods/ftblibrary/ui/misc/SimpleToast.java

```diff
@@ -1,33 +1,27 @@
 package dev.ftb.mods.ftblibrary.ui.misc;
 
 import com.mojang.blaze3d.platform.Lighting;
-import com.mojang.blaze3d.systems.RenderSystem;
 import dev.ftb.mods.ftblibrary.icon.Icon;
 import dev.ftb.mods.ftblibrary.icon.Icons;
 import dev.ftb.mods.ftblibrary.ui.GuiHelper;
 import net.minecraft.client.gui.GuiGraphics;
 import net.minecraft.client.gui.components.toasts.Toast;
 import net.minecraft.client.gui.components.toasts.ToastComponent;
-import net.minecraft.client.renderer.GameRenderer;
 import net.minecraft.client.sounds.SoundManager;
 import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
 import net.minecraft.util.Mth;
 
-/**
- * @author LatvianModder
- */
 public class SimpleToast implements Toast {
+	private static final ResourceLocation BACKGROUND_SPRITE = new ResourceLocation("toast/advancement");
 	private boolean hasPlayedSound = false;
 
 	@Override
 	public Visibility render(GuiGraphics graphics, ToastComponent gui, long delta) {
 		GuiHelper.setupDrawing();
 		var mc = gui.getMinecraft();
 
-		RenderSystem.setShader(GameRenderer::getPositionTexShader);
-		RenderSystem.setShaderTexture(0, TEXTURE);
-		RenderSystem.setShaderColor(1F, 1F, 1F, 1F);
-		graphics.blit(TEXTURE, 0, 0, 0, 0, 160, 32);
+		graphics.blitSprite(BACKGROUND_SPRITE, 0, 0, 160, 32);
```

Comment:
This felt like the right modification but it seems to still not be working so?....